zerver/views/reactions.py

```diff
@@ -22,14 +23,17 @@ def create_historical_message(user_profile: UserProfile, message: Message) -> No
                                message=message,
                                flags=UserMessage.flags.historical | UserMessage.flags.read)
 
+# transaction.atomic is required for the use of select_for_update in
+# access_message when lock_message or lock_usermessage is passed.
```

Comment:
If we want these comments, they should be a bit more concise imo - they can just say that we run the function in a transaction to be a able to hold a FOR UPDATE lock throughout its course. (no need to elaborate on the implementation details of how `access_message` is called and what its arguments are called.

Reply:
Is this fine?
```
# transaction.atomic is required since we use FOR UPDATE queries in access_message
```